pandas/core/indexing.py

```diff
@@ -2692,8 +2692,9 @@ def is_nested_tuple(tup, labels):
 
 def is_list_like_indexer(key):
     # allow a list_like, but exclude NamedTuples which can be indexers
-    return is_list_like(key) and not (isinstance(key, tuple) and
-                                      type(key) is not tuple)
+    return (is_list_like(key)
+            and not (isinstance(key, tuple) and type(key) is not tuple)
+            and not np.array(key).ndim == 0)
```

Comment:
You're really just checking if you have a scalar.  Is it possible to use (and potentially modify) `is_scalar` from `pandas.core.dtypes.inference`?

Reply:
Possibly! I'll look into that more tomorrow.